src/Geocode/GeocodeResultsFactory.php

```diff
@@ -12,15 +13,24 @@
 
 class GeocodeResultsFactory
 {
+    /** @var LoggerInterface */
+    private $logger;
+
+    public function __construct(LoggerInterface $logger)
+    {
+        $this->logger = $logger;
+    }
+
     /**
+     * @param array $response
      * @return GeocodeResult[]
      */
     public function create(array $response) : array
     {
         $results = [];
 
         if (empty($response['results'])) {
```

Comment:
si tu peux juste remplacer ça, c'est plus performant comme ça
```suggestion
        if ($response['results'] === []) {
```